squirrel.frontier/src/main/java/org/dice_research/squirrel/queue/scorecalculator/IUriScoreCalculator.java

```diff
@@ -0,0 +1,16 @@
+package org.dice_research.squirrel.queue.scorecalculator;
+
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+
+/**
+ *  Interface for getting duplicity score of a Uri.
+ */
+public interface IUriScoreCalculator {
```

Comment:
Is it really a duplicity score as described in the javadoc comment? The interface itself simply defines a method to calculate a score for a given URI. So why do you want to reduce its usage to the use case of searching for duplicates?